frontend/.env.production

```diff
@@ -11,4 +11,5 @@ SENDY_API_KEY=
 SENDY_API_URL=
 SENDY_LIST_ID=
 
-API_URL=http://api.simpler.grants.gov
+API_URL=https://api.simpler.grants.gov
+AUTH_LOGIN_URL=https://api.simpler.grants.gov/v1/users/login
```

Comment:
Yeah, I guess we have the API one to point to that the behavior is "as expected" but that likely also reinforces that they don't need to be here?

Reply:
It appears this gets picked up at build time, even though it's in a client component, because... I'm not sure how or why? Which also makes me question how staging is not finding Prod, as at that build it definitely doesn't have access to the SSM stuff (and I can't double check what SSM has for Stage cause I can't SSO into AWS right now).

Reply:
I've unfortunately fought this stuff a bunch with the new relic work. It might be good to talk this out if folks have time